src/clients/virtualization.rb

```diff
@@ -76,13 +76,17 @@ def main
     def CheckConfiguration
       ret = true
       is_s390 = false
+      is_aarch64 = false
 
       # check whether VM can be started (cannot start a vm using UML)
       return false if VirtConfig.isUML
 
       # s390 is technical preview and we only fully support x86_64
       if Arch.s390_64 == true
         is_s390 = true
+      # aarch64 is technical preview and we only fully support x86_64
+      elsif Arch.aarch64 == true
```

Comment:
This change looks quite pointless at first sight. It seems to set a local variable that is not used for any purpose. Am I overlooking something?